string Class
float64 probability
int32 xmin
int32 ymin
int32 xmax
int32 ymax
int32[] encoded_pixels
int32 pixel_size